setup.py

```diff
@@ -21,6 +21,15 @@ def main():
     with open(info_file) as infofile:
         exec(infofile.read(), globals(), ldict)
 
+    try:
+        import versioningit
+    except ImportError:
+        # versioningit isn't installed; assume we're building a Debian package
```

Comment:
well -- not just Debian but any packaging environment (e.g. conda-forge) which doesn't have versioningit installed and/or not using PEP 518

Reply:
We should be using `versioningit` when building in conda-forge; the lack thereof is the cause of things like https://github.com/dandi/dandi-schema/issues/135.  The comment should not be generalized to other non-PEP 517 environments, as they should be discouraged as much as possible.

Reply:
well -- silent discouragement is not a discouragement, it is "allowance" ;-)  Similarly I consider the fact that packages silently install with degenerate version -- it would be better if they failed.  So such plugs as the diff here now is generally needed not only in heudiconv and not only on Debian but IMHO in every versiongit using package.  But I will not fight over a comment here.